src/services/query-builder.js

```diff
@@ -35,6 +35,9 @@ function QueryBuilder(model, opts, params) {
           includes.push({
             model: association.target.unscoped(),
             as: association.associationAccessor,
+            // NOTICE: For performance reasons, we only request the keys
+            //         as they're the only needed fields for the interface
+            attributes: [association.sourceKey, association.targetKey].filter(Boolean),
```

Comment:
You cannot exclude everything, as this will break the reference field:
With your modification:
<img width="182" alt="Capture d’écran 2020-08-04 à 15 33 28" src="https://user-images.githubusercontent.com/3263749/89300337-5937e680-d668-11ea-8925-f68c1146bec5.png">
Before:
<img width="184" alt="Capture d’écran 2020-08-04 à 15 35 41" src="https://user-images.githubusercontent.com/3263749/89300340-5a691380-d668-11ea-8857-28d1bbe8a51b.png">

All the fields you need are sent by the front in the query params:
<img width="916" alt="Capture d’écran 2020-08-04 à 15 38 35" src="https://user-images.githubusercontent.com/3263749/89300490-98663780-d668-11ea-8084-30e45eea76a7.png">

PS: I don't think you need the `targetKey` and `sourceKey`

Reply:
Thanks for the test. I had to do more work on the backend side to get all the requested fields.

Regarding `targetKey` and `sourceKey`, if I remove them, it does not work either.

The new fix has to be tested with https://github.com/ForestAdmin/forestadmin/pull/2157